lib/bootstrap_builders/class_attribute_handler.rb

```diff
@@ -9,6 +9,14 @@ def add(class_argument)
     @classes += convert_to_array(class_argument)
   end
 
+  def remove(class_argument)
+    if class_argument.is_a?(Array)
+      @classes = @classes - class_argument
```

Comment:
I believe you can write this just a tiny bit shorter. Do you mind giving it a try?

```ruby
@classes -= class_argument
```